models/contrib/nish/curve.py

```diff
@@ -56,16 +56,17 @@ class CurvePoolPeggingInfo(Contract):
     chi: float
     ratio: float
 
+
 @Model.describe(slug="contrib.curve-get-pegging-ratio",
-                version="1.0",
+                version="1.1",
```

Comment:
is this from your merge?

Reply:
or something else?

Reply:
This is nish's code that I am reviewing. Correcting pylint complaints